src/sonic-yang-models/yang-models/sonic-ssh-server.yang

```diff
@@ -40,6 +45,20 @@ module sonic-ssh-server {
                         }
                     }
                 }
+                leaf inactivity_timeout {
+                    description "inactivity timeout (in minutes), 0 means no timeout";
+                    default 15;
```

Comment:
>15 [](http://example.com/codeflow?start=28&length=2)

How do you pick the default value? Is it the linux default behavior in old image without this feature?

Reply:
@qiluo-msft hi, yes, this is old defaults in sonic